lib/pure/typetraits.nim

```diff
@@ -10,17 +10,18 @@
 ## This module defines compile-time reflection procs for
 ## working with types
 
-proc name*(t: typedesc): string {.magic: "TypeTrait".}
+proc name*(t: typedesc): string =
   ## Returns the name of the given type.
   ##
+  ## As of Nim v0.18.0 this is just an alias for `$` which is defined in
```

Comment:
Before merging, change it to 0.19.2 or 0.20, depending on when this will be introduced.